proparse/src/main/java/org/prorefactor/treeparser01/FrameStack.java

```diff
@@ -189,7 +189,7 @@ private Frame initializeFrame(Frame frame, Block currentBlock) {
    * 
    * @see org.prorefactor.core.JPNode#getFieldContainer().
    */
-  FieldLookupResult inputFieldLookup(FieldRefNode fieldRefNode, SymbolScope currentScope) {
+  FieldLookupResult inputFieldLookup(FieldRefNode fieldRefNode, TreeParserSymbolScope currentScope) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 21 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS3776)